examples/src/main/java/org/apache/spark/examples/mllib/JavaGaussianMixtureExample.java

```diff
@@ -44,8 +44,9 @@ public static void main(String[] args) {
         public Vector call(String s) {
           String[] sarray = s.trim().split(" ");
           double[] values = new double[sarray.length];
-          for (int i = 0; i < sarray.length; i++)
+          for (int i = 0; i < sarray.length; i++) {
```

Comment:
Oh, I fix this since it's the error from one 'Checkstyle' (lint-java).
Up to now, this PR does not include `stop()` context discussed in emails.